pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -669,19 +669,7 @@ func (h *Helper) ServerPrereleaseVersion() (semver.Version, error) {
 		return semver.Version{}, fmt.Errorf("did not find version in command output")
 	}
 
-	// The OSE version may have > 4 parts to the version string
-	// We'll only take the first 3
-	parts := strings.Split(versionStr, ".")
-	if len(parts) > 3 {
-		versionStr = strings.Join(parts[:3], ".")
-	}
```

Comment:
We should look for a 4th number, like try to match `/([0-9]+).([0-9]+).([0-9]+).([0-9]+)(.*)/` and if it does, then replace with `\1.\2.\3\5`